### ISSUE TITLE
Log: defined with -nargs=+, but handles only one

### ISSUE THREAD

**Issue by blueyed** [junegunn/vader.vim#89]
Defined in https://github.com/junegunn/vader.vim/blob/85a04a53857453b038fb89648622d85b48da60fd/autoload/vader.vim#L171, but only one arg is handled:
```vim
function! vader#log(msg)
  let msg = type(a:msg) == 1 ? a:msg : string(a:msg)
  call vader#window#append('> ' . msg, s:indent)
endfunction
```

I have tried `Log foo bar` several times, resulting in an error.

I think it should be `vader#log(...)`, looping over `a:000`, and add the result joined by commas?! (so each `Log` call is a single line still)

**Comment by junegunn** [comment]
You're right. I don't remember why I put `-nargs=+`, it was probably an oversight. How about if we just change it to `-nargs=1`? Do you prefer multi-arg version?

**Comment by blueyed** [comment]
I think a multi-arg version is useful, if it joins the args into one line.
